crates/codegen/parser/runtime/src/napi_interface/cst.rs

```diff
@@ -83,10 +83,43 @@ impl RuleNode {
             .into()
```

Comment:
nit: I think this is missing `catch_unwind` here as well?

Reply:
Huh, I would've expected it to properly propagate if the proxied to function already uses `catch_unwind` but I've double checked and in fact the error is not propagated. Good catch! I'll fix this.